database/neo4j/neo4j.go

```diff
@@ -119,7 +124,7 @@ func (n *Neo4j) Run(migration io.Reader) (err error) {
 	}()
 
 	if n.config.MultiStatement {
-		statements := bytes.Split(body, []byte{';'})
+		statements := bytes.Split(body, []byte{StatementSeparator})
```

Comment:
Casting to a `[]byte` should be sufficient. Or make the package level constant/variable a `[]byte`